openshift/4.0/templates/api/deploy-alt-routes.yaml

```diff
@@ -100,6 +100,23 @@ parameters:
     description: "The Common Hosted Email Service password."
     required: true
 
+  - name: MY_LTSA_USERNAME
+    displayName: "My Ltsa Username."
+    description: "The LTSA username."
+    required: true
```

Comment:
For CITZ PIMS, I wonder if we should not make these required.  It will just throw credential errors if they attempt to use the endpoints for LTSA.

Reply:
Is that a realistic concern?